crates/biome_cli/tests/snapshots/main_cases_config_extends/extends_config_ok_linter_not_formatter.snap

```diff
@@ -52,7 +52,7 @@ check ━━━━━━━━━━━━━━━━━━━━━━━━
 ```block
 test.js:1:1 lint/suspicious/noDebugger  FIXABLE  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 
-  × This is an unexpected use of the debugger statement.
+  ! This is an unexpected use of the debugger statement.
```

Comment:
This is the only test case where I think the original snapshot was actually incorrect. The config already specified it should be a warning, but it looks like it was previously still reported as an error.